packages/gatsby-plugin-hoofd/gatsby-ssr.js

```diff
@@ -2,14 +2,18 @@ import { createElement } from 'react';
 import { toStatic } from 'hoofd';
 
 export const onRenderBody = ({ setHeadComponents, setHtmlAttributes }) => {
-  const { title, metas, lang, links } = toStatic();
+  const { title, metas, lang, links, amp, ampScript } = toStatic();
 
   if (lang) {
-    setHtmlAttributes({ lang });
+    setHtmlAttributes({ lang, amp });
```

Comment:
It is also possible to create an amp document without a lang attribute, does this account for this condition?

(I don't see any tests so I couldn't confirm)

Reply:
Yes, when lang is undefined it shouldn't add it